src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,208 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4; // aka 16
+    private static final int MAXIMUM_CAPACITY = 1 << 30;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int TREEIFY_THRESHOLD = 9;
+    private static final int MIN_TREEIFY_CAPACITY = 64;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+    private final float loadFactor;
+
+    public MyHashMap() {
+        this.loadFactor = DEFAULT_LOAD_FACTOR; // all other fields defaulted
+    }
+
+    private static int hash(Object key) {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private int hash(Object key) {
```